src/trustpilot-api.ts

```diff
@@ -1,13 +1,15 @@
 import { AccessProvider } from './access-provider';
 import { ITrustpilotApiConfig } from './models';
 import { RequestHelper } from './request-helper';
+import * as http from 'http';
+import * as https from 'https';
 
 export class TrustpilotApi {
   private requestHelper: RequestHelper;
 
-  constructor(config: Readonly<ITrustpilotApiConfig>) {
+  constructor(config: Readonly<ITrustpilotApiConfig>, agent?: http.Agent | https.Agent) {
```

Comment:
do we ever allow `http` traffic or should we only accept `https.Agent`?

Reply:
Most of the time it will be Https. I've added the Http just for the sake of opportunity.